src/RequestList.tsx

```diff
@@ -7,10 +7,57 @@ import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import { isCarRequest, messageFromRequest } from "./util";
+import FiberManualRecordIcon from '@mui/icons-material/FiberManualRecord';
+
+function getRequestStatus(request: Request) {
```

Comment:
observation: I think this logic is identical to the logic above
request: please factor this out into a library function